sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/xunit/CxxXunitSensor.java

```diff
@@ -110,21 +110,6 @@ protected String reportPathKey() {
   public Class<?> dependsUponCoverageSensors() {
     return CoverageExtension.class;
   }
-  
-  /**
-   * {@inheritDoc}
-   */
-  @Override
-  public boolean shouldExecuteOnProject(Project project) {
-    if (conf.hasKey(reportPathKey())) {
-      if (!conf.getBoolean(PROVIDE_DETAILS_KEY)) {
-        return !project.isModule();
-      } else {
-        return fs.hasFiles(fs.predicates().hasLanguage(CxxLanguage.KEY));
```

Comment:
Not sure if removing shouldExecuteOnProject is a good idea? Think it should test at least ... hasLanguage(CxxLanguage.KEY)?